docs/basic-usage.ipynb

```diff
@@ -91,7 +91,7 @@
     "\n",
     "A travel time matrix is a data set detailing the travel costs (e.g., time) from each point to each point in a set of origins and destinations in a study area. \n",
     "\n",
-    "For the sake of this exercise, we will create a regular grid of points covering parts of Helsinki."
+    "For the sake of this exercise, we have prepared a grid of points covering parts of Helsinki. The point dataset has been obtained from [Helsinki Region Environmental Services](https://www.hsy.fi/en/environmental-information/open-data/avoin-data---sivut/population-grid-of-helsinki-metropolitan-area/) (HSY), and it also contains information about the number of citizens living within a 250 meter grid cells. "
```

Comment:
Could we add ’under a Creative Commons By Attribution 4.0 license’, and make it ’residents of each cell’ (they (we!) are not necessarily citizens ;) )

Reply:
Sure, will modify accordingly.